pkg/controller/disruption/disruption.go

```diff
@@ -452,9 +478,21 @@ func (dc *DisruptionController) trySync(pdb *policy.PodDisruptionBudget) error {
 		return err
 	}
 
-	currentHealthy := countHealthyPods(pods)
-	err = dc.updatePdbSpec(pdb, currentHealthy, desiredHealthy, expectedCount)
+	currentTime := time.Now()
+	disruptedPods, recheckTime := buildDisruptedPodMap(pods, pdb, currentTime)
```

Comment:
I meant a comment about the clock skew issue if the controller manager and API server are running on different nodes.